indigo_app/views/works.py

```diff
@@ -105,6 +105,7 @@ class EditWorkView(WorkViewBase, UpdateView):
     form_class = WorkForm
     prefix = 'work'
     permission_required = ('indigo_api.change_work',)
+    tab = 'overview'
```

Comment:
if we nuked this line it would make it more obvious that the Overview is somewhere you currently aren't, which I like?
![image](https://user-images.githubusercontent.com/32566441/68870526-b3bc9b80-0703-11ea-9276-ca3e79596068.png)

Reply:
nice.

Reply:
done.